src/cf-core/src/cf-base.less

```diff
@@ -4,7 +4,12 @@
    ========================================================================== */
 
 //
-// Webfonts
+// Webfont mixins
+//
+// WARNING: These mixins are deprecated and will be removed in CFv5 due to
```

Comment:
You could include a `TODO` like https://github.com/cfpb/capital-framework/blob/61216e04e78e293ff2a9e3075c9804dd91484c72/src/cf-core/src/cf-utilities.less#L55

Reply:
Done in d82c3a0401457a22301dfffb2dcaac037021e948